src/tracing/types.rs

```diff
@@ -57,6 +67,12 @@ pub struct CallTrace {
     pub status: InstructionResult,
     /// Opcode-level execution steps.
     pub steps: Vec<CallTraceStep>,
+    /// Optional decoded label for the call.
+    pub decoded_label: Option<String>,
+    /// Optional decoded return data.
+    pub decoded_return_data: Option<String>,
+    /// Optional decoded call data.
+    pub decoded_call_data: Option<DecodedCallData>,
```

Comment:
could we bundle this in one container type for decoding?

Reply:
Done, did the same for Log, removed the `decoded_*` prefix now that it has container types